controllers/apim/ratelimitpolicy_controller.go

```diff
@@ -22,21 +22,20 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/go-logr/logr"
+	"github.com/gogo/protobuf/types"
+	"github.com/kuadrant/limitador-operator/api/v1alpha1"
```

Comment:
nit: can you please group relevant ones together?

Reply:
Well, this is some personal option that I like. I always add imports grouped in 3 groups: A) builtin imports B)packages not included in the current module C) packages from within the current module.

What is your definition of "relevant"? All of them are relevant, it will not compile if you remove them :)

Reply:
haha, I was grouping 1) std 2) k8s 3) kuadrant 4) rest but it's fine as in 3 groups as well. :+1: